src/vi/nvop-sub.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gpu-operator-certified
+  namespace: nvidia-gpu-operator
+spec:
+  channel: "{CHANNEL}"
+  installPlanApproval: Manual
+  name: gpu-operator-certified
+  source: certified-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: "{CSV}"
```

Comment:
I believe startingCSV can be dropped. Also - we probably want to configure the channel from an environment variable set from the deployment found here: https://github.com/Azure/maximo/blob/main/src/azure/cloud-init.yaml

Similar to export masChannel="${masChannel}" so that the user can configure this during deployment.

Reply:
can install plan be set to automatic?

installPlanApproval: Automatic

Reply:
install plan updated to automatic; leaving both for now as they are seperate values.

Question: do we want this to be a parameter, or have the installer query each time to figure out the correct version? Leaving as is but can change.

https://github.com/Azure/maximo/pull/15/commits/5c84134801f4ea715f2c51ab83e3b7ade74cfc38